docs/tech.md

```diff
@@ -164,7 +164,7 @@ Jars are heuristically detected as Minecraft mods or plugins as follows:
 
 ## Stage3 (`unobf-client.jar`)
 
-Around 2023-06-07 14:20 UTC the stage3 client jar was seemingly accidentally replaced with an unobfuscated version. You can find the archive here: https://github.com/clrxbl/NekoClient
+Around 2023-06-07 14:20 UTC the Stage-3 client jar was seemingly accidentally replaced with an unobfuscated version. You can find the archive here: https://github.com/clrxbl/NekoClient
```

Comment:
Your PR summary says you aren't touching these, so i'd revert this for consistency (nothing else in the doc uses a dash)

Reply:
Extended the PR to fix this mistake (actually the other places simply didn't have a space, otherwise you're right).

I only kept the Heading "Stage3" the same, in case someone created an anchor link to that.